web/vtadmin/package.json

```diff
@@ -33,23 +31,22 @@
     "react-toastify": "^8.1.0",
```

Comment:
@notfelineit I'm seeing also a following message:

```
npm WARN deprecated react-flow-renderer@10.3.17: react-flow-renderer has been renamed to reactflow, please use this package from now on https://reactflow.dev/docs/guides/migrate-to-v11/
```

Should we also update all dependencies here in general? Or do that in a separate PR then?

Reply:
I'm thinking we can do that in a separate PR! Should be easy to upgrade now we're on Vite 😄